src/Orchard/ContentManagement/DefaultContentManager.cs

```diff
@@ -722,7 +722,7 @@ public class DefaultContentManager : IContentManager {
 
         // Insert or Update imported data into the content manager.
         // Call content item handlers.
-        public void Import(XElement element, ImportContentSession importContentSession) {
+        public void Import(XElement element, ImportContentSession importContentSession, Action<ContentItem> importItemCallback = null) {

```

Comment:
Nope. Please create a new `IEventHandler` for this event. There might be others.